.buildbot.sh

```diff
@@ -2,5 +2,31 @@
 # vim: expandtab sts=0 sw=4 smarttab
 set -e
 
-echo "Test buildbot" 
+ROOT_DIR=$(pwd)/..
 
+# arg-1 : Root directory
+# arg-2 : cmake toolchain file
+function build_boehmGC()
+{
+    ARCH=$(echo ${2} | cut -d '.' -f 1)
+    SRC_DIR=${1}/build # build/source directory name used for all repos
```

Comment:
Isn't this literally *not* the source directory -- it's the build directory?

Reply:
This is where the source is placed by buildbot for all the repositories.

Reply:
I think we can remove `SRC_DIR`.